doc/DeveloperGuide.md

```diff
@@ -60,6 +60,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | login with social media account | automatically sync contact information
+`* *` | user | navigate an intuitive application | get anything I want done with minimal effort
```

Comment:
This sounds like a non-functional requirement (i.e. not suitable to be captured as a user story)